cv32/sim/xrun/probe.tcl

```diff
@@ -0,0 +1,5 @@
+database -open waves -default -incsize 2G
```

Comment:
Same comment: maybe these should be in `cv32/sim/tools/xrun` (not `cv32/sim/xrun`).

Reply:
Yep...will amend.